permission_handler_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.5.0
+
+*Added support for app tracking transparency permission.
```

Comment:
nit: would be nice to add a space between the asterisks and the word "Added", otherwise the MarkDown language won't recognise it as a bullet.

```suggestion
* Added support for app tracking transparency permission.
```

Should also be fixed with version 3.4.0.